plugins/modules/foreman_auth_source_ldap.py

```diff
@@ -180,6 +185,7 @@ def main():
             ldap_filter=dict(),
             locations=dict(type='entity_list', flat_name='location_ids'),
             organizations=dict(type='entity_list', flat_name='organization_ids'),
+            use_netgroups=dict(type='bool', default=False),
```

Comment:
```suggestion
            use_netgroups=dict(type='bool'),
```

If you omit the `default`, the param will only be sent to the server when the user has explicitly set it, thus allowing the server to decide on a default.

Also, the APIdoc says "applicable only when server_type is posix or free_ipa", so we might want to add a guard somehow not to set that on AD connections.

Reply:
I'm not seeing an easy way to add the guard you suggested.  I need, I think, something close to the opposite of the required_if call.  Any suggestions here?

Reply:
Nevermind, I think I figured it out.